a3p-integration/proposals/a:upgrade-next/use.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-# UNTIl this is upstream https://github.com/Agoric/agoric-3-proposals/issues/40
+# UNTIL this is upstream https://github.com/Agoric/agoric-3-proposals/issues/40
```

Comment:
Arf this is still around? I guess I need to land https://github.com/Agoric/agoric-sdk/pull/8886

Reply:
the issue references by the UNTIL is closed, so the sed should be removed. Good to retain the file for documentation as in 8866. You could just wait for that PR which replaces the file contents, but know that landing this 1char change will cause that one to have merge conflicts.